core/contracts/oracle/implementation/Voting.sol

```diff
@@ -312,7 +312,7 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
      * Commits can be changed.
      * @param identifier uniquely identifies the committed vote. EG BTC/USD price pair.
      * @param time unix timestamp of the price being voted on.
-     * @param hash keccak256 hash of the price you want to vote for and a `int salt`.
```

Comment:
nit: is it better to say "price, salt, and address" if that's the order the params are packed in?

Reply:
Yeah I can make the comment more specific that the hash must include "price, salt, address"